src/server/generic_family.cc

```diff
@@ -397,7 +397,9 @@ OpStatus Renamer::UpdateDest(Transaction* t, EngineShard* es) {
         dest_it->second = std::move(pv_);
       }
       dest_it->second.SetExpire(has_expire);  // preserve expire flag.
-      db_slice.UpdateExpire(t->GetDbIndex(), dest_it, src_res_.expire_ts);
+      if (!has_expire && src_res_.expire_ts) {
+        db_slice.AddExpire(t->GetDbIndex(), dest_it, src_res_.expire_ts);
```

Comment:
i suspect it's wrong. please revert this change (unfortunately)

Reply:
sure